<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Reinhold Kainhofer &lt;reinhold@kainhofer.com&gt;</author>
 <comment>Configuration page for the print journal mode.</comment>
 <class>CalPrintJournalConfig_Base</class>
 <widget class="QWidget" name="CalPrintJournalConfig_Base">
  <layout class="QVBoxLayout">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Print journal options:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mDateRangeGroup">
     <property name="title">
      <string>Date &amp;&amp; Time Range</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="mAllJournals">
        <property name="text">
         <string>&amp;All journal entries</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mRangeJournals">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>Date &amp;range:</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="mDateRangeLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>24</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="mFromDateLabel">
          <property name="toolTip">
           <string>Starting date for the print</string>
          </property>
          <property name="whatsThis">
           <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;End date&lt;/i&gt; option. This option is used to define the start date.</string>
          </property>
          <property name="text">
           <string>&amp;Start date:</string>
          </property>
          <property name="buddy">
           <cstring>mFromDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KDateComboBox" name="mFromDate">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="toolTip">
           <string>Starting date for the print</string>
          </property>
          <property name="whatsThis">
           <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;End date&lt;/i&gt; option. This option is used to define the start date.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mToDateLabel">
          <property name="toolTip">
           <string>Ending date for the print</string>
          </property>
          <property name="whatsThis">
           <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;Start date&lt;/i&gt; option. This option is used to define the end date.</string>
          </property>
          <property name="text">
           <string>&amp;End date:</string>
          </property>
          <property name="buddy">
           <cstring>mToDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KDateComboBox" name="mToDate">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="toolTip">
           <string>Ending date for the print</string>
          </property>
          <property name="whatsThis">
           <string>If you want to print more days at once, you can define a range of dates with this option and the &lt;i&gt;Start date&lt;/i&gt; option. This option is used to define the end date.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mSecurity">
     <property name="title">
      <string>Security Exclusions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="mExcludeConfidential">
        <property name="toolTip">
         <string>Check this option to exclude items that have their Access level set to &#8220;Confidential&#8221;</string>
        </property>
        <property name="text">
         <string>Exclude c&amp;onfidential</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mExcludePrivate">
        <property name="toolTip">
         <string>Check this option to exclude items that have their Access level set to &#8220;Private&#8221;</string>
        </property>
        <property name="text">
         <string>Exclude pri&amp;vate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGeneralGroup">
     <property name="title">
      <string comment="@title general print settings">General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="mPrintFooter">
        <property name="toolTip">
         <string>Print a datetime footer on each page</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to print a small footer on each page that contains the date of the print.</string>
        </property>
        <property name="text">
         <string>Print &amp;Footer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mRangeJournals</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFromDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRangeJournals</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFromDateLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRangeJournals</sender>
   <signal>toggled(bool)</signal>
   <receiver>mToDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRangeJournals</sender>
   <signal>toggled(bool)</signal>
   <receiver>mToDateLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
